crypto/bio/bf_readbuff.c

```diff
@@ -197,11 +199,19 @@ static long readbuffer_ctrl(BIO *b, int cmd, long num, void *ptr)
             ret = BIO_ctrl(b->next_bio, cmd, num, ptr);
         }
         break;
+    case BIO_CTRL_PUSH:
+    case BIO_CTRL_POP:
     case BIO_CTRL_DUP:
     case BIO_CTRL_FLUSH:
-        ret = 1;
         break;
+
+    case BIO_CTRL_SET_PREFIX:
+    case BIO_CTRL_SET_INDENT:
+    case BIO_CTRL_GET_INDENT:
```

Comment:
These seem superfluous now...

Reply:
Right - my original aim was to raise an error only for those,
but as you pointed out in https://github.com/openssl/openssl/pull/18434#pullrequestreview-1046558812, the cmd `default` case should handle all commands not suitable for the BIO at hand.
So removed again those three explicit cases for all BIO implementations where I had added them.